config/dispatcher/dispatcher.ex

```diff
@@ -84,4 +84,8 @@ defmodule Dispatcher do
   match "/*_", %{ layer: :not_found } do
     send_resp( conn, 404, "Route not found.  See config/dispatcher.ex" )
   end
+
+  match "/holiday-counters/*path", %{ layer: :services, accept: %{ json: true } } do
```

Comment:
For readability it's best to not add the rule below the catch all. Best to put this one near the other resource rules like

Reply:
Probably missing dispatcher rules for concept/concept-scheme?